test/gpu_tests.jl

```diff
@@ -0,0 +1,373 @@
+using Test
+using KernelAbstractions
+using CUDAKernels
+
+import CloudMicrophysics
+import CLIMAParameters
+import Thermodynamics
+
+const CM = CloudMicrophysics
```

Comment:
```suggestion
const CM = CloudMicrophysics
const CM0M = CloudMicrophysics.Microphysics_0M
const CM1M = CloudMicrophysics.Microphysics_1M
```
I might add these aliases to see if you could squeeze out a few more lines of code.